demos/storybook/stories/dropdown-toolbar/with-custom-menu.stories.ts

```diff
@@ -0,0 +1,23 @@
+import { text } from '@storybook/addon-knobs';
+import * as Colors from '@pxblue/colors';
+import { action } from '@storybook/addon-actions';
+
+export const withCustomMenu = (): any => ({
+    template: `
+       <pxb-dropdown-toolbar [title]="title" [subtitle]="subtitle">
+        <ng-container pxb-toolbar-menu>
+            <mat-nav-list>
+                <pxb-info-list-item (click)="click()"><mat-icon [style.color]="colors.blue[500]" pxb-icon>business</mat-icon><span pxb-title>Atlanta</span></pxb-info-list-item>
```

Comment:
These look cool! Can we use the dense variant and also update the subtitle when they are selected like the other stories?